### PR TITLE
Add `global.json` and other missing paths to `CODEOWNERS`

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
This PR contributes to:

- #5009

With this PR merged, the CODEOWNERS file should have 100% coverage of all files and directories currently in the repository.

### CODE REVIEW

**Comment by konrad-jamrozik** [comment]
@weshaggard should other files in root dir also be added to CODEOWNERS?

image

**Comment by weshaggard** [comment]
Yes we should probably include all the root files perhaps \* works for that.

**Comment by konrad-jamrozik** [comment]
@weshaggard I added rules to cover everything in repo:

https://github.com/Azure/azure-sdk-tools/pull/5044/files

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,16 @@
+################
+# Misc
+################
+/*                               @azure/azure-sdk-eng
+/.azure-pipelines/               @azure/azure-sdk-eng
+/.devcontainer/                  @azure/azure-sdk-eng
+/artifacts/                      @azure/azure-sdk-eng
```
Comment (weshaggard) [inline .github/CODEOWNERS:7]: This is a generated path and not committed in the repo.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+/config/                         @azure/azure-sdk-eng
+/doc/                            @azure/azure-sdk-eng
+/scripts/                        @azure/azure-sdk-eng
```
Comment (weshaggard) [inline .github/CODEOWNERS:10]: Some of these other paths probably have multiple owners but we can start out our team and then split out as needed.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+/src/                            @azure/azure-sdk-eng
+/web/                            @azure/azure-sdk-eng
```
Comment (weshaggard) [inline .github/CODEOWNERS:12]: Some things like these (src/web) we also eventually want to reorganize under tools directory.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,3 +1,16 @@
+################
+# Misc
+################
+/*                               @azure/azure-sdk-eng
+/.azure-pipelines/               @azure/azure-sdk-eng
```
Comment (weshaggard) [inline .github/CODEOWNERS:5]: Eventually want to move this under eng/pipelines.

File: .github/CODEOWNERS (hunk 2 above)
Comment (weshaggard) [inline .github/CODEOWNERS:10]: This scripts folder should be moved under eng/scripts at some point.

**Comment by weshaggard** [comment]
@konrad-jamrozik I'm good with what you have except for the artifacts would should be removed. The other comments are more brain dumping from me of more clean-up I would like to do in the tools repo at some point.

**Review by weshaggard — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (konrad-jamrozik) [reply .github/CODEOWNERS:13]: Created:
- #5056

File: .github/CODEOWNERS (hunk 4 above)
Comment (konrad-jamrozik) [reply .github/CODEOWNERS]: Created:
- #5056

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+/.devcontainer/                  @azure/azure-sdk-eng
+/artifacts/                      @azure/azure-sdk-eng
+/config/                         @azure/azure-sdk-eng
```
[the remark this answers is not in the document]
Comment (konrad-jamrozik) [reply .github/CODEOWNERS]: Created:
- #5056

File: .github/CODEOWNERS (hunk 2 above)
Comment (konrad-jamrozik) [reply .github/CODEOWNERS:11]: Created:
- #5056

**Pull request merged**